src/markdownIt/utils.ts

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2023 Avaiga Private Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+import { l10n, window } from "vscode";
+import path from "path";
+import { existsSync, readFileSync } from "fs";
+import { getLog } from "../gui/logging";
+
+// export const camelize = (text: string): string => {
+//     const a = text.toLowerCase().replace(/[-_\s.]+(.)?/g, (_, c) => (c ? c.toUpperCase() : ""));
+//     return a.substring(0, 1).toLowerCase() + a.substring(1);
+// };
+
+export const camelize = (text: string): string => {
+    return text.replace(/^([A-Z])|[\s-_]+(\w)/g, function (match, p1, p2, offset) {
+        if (p2) {
+            return p2.toUpperCase();
+        }
+        return p1.toLowerCase();
+    });
+};
+
+export const parseMockData = (value: string): string => {
+    if (!window.activeTextEditor?.document) {
+        return value;
+    }
+    const document = window.activeTextEditor.document;
+    const potentialMockFile = path.join(path.dirname(document.uri.fsPath), path.parse(document.fileName).name + ".mock.json");
```

Comment:
Should use workspace.fs instead of fs
and workspace.asRelativePath & Uri.joinPath ...
example:
```
const content = await workspace.fs.readFile(Uri.file(schemas["taipy.core"]));
validationSchema = JSON.parse(Buffer.from(content).toString('utf8'));
```